proto/tendermint/types/types.proto

```diff
@@ -125,11 +125,8 @@ message EvidenceList {
   repeated Evidence evidence = 1 [(gogoproto.nullable) = false];
 }
 
-message Messages {
-  repeated Message messages_list = 1;
-}
-
-message Message {
+// Blob defines an indivisible chunk of data that is posted on chain.
```

Comment:
[optional]

I like the desire to include a doc comment above this critical type. I think "indivisible" may give readers with knowledge of the implementation details an inaccurate understanding because the blob may be split into multiple shares if it can't fit in one share. I may be reading too deeply into this so I think the current comment is fine but if you agree then here's a proposal

```suggestion
// Blob defines a chunk of data that is attributed to a namespace and under
// usual circumstances, ends up published on-chain.
```